tools/accuracy_checker/configs/googlenet-v1-tf.yml

```
@@ -23,15 +23,15 @@ models:
       - framework: dlsdk
         tags:
           - FP32
-        model:   public/googlenet-v1/tf/FP32/googlenet-v1.xml
-        weights: public/googlenet-v1/tf/FP32/googlenet-v1.bin
+        model:   public/googlenet-v1-tf/tf/FP32/googlenet-v1.xml
```

these is not `tf` folder inside,
name format:  public/<model_name>/<precision>/<model_name>.xml

```suggestion
        model:   public/googlenet-v1-tf/FP32/googlenet-v1-tf.xml
```